src/content/docs/logs/logs-context/get-started-logs-context.mdx

```diff
@@ -101,6 +106,39 @@ Our log ingest pipeline automatically masks credit cards, Social Security number
 
 You can also create custom rules to mask or hash sensitive data in your logs with our obfuscation feature. This is critical when it is impractical or impossible to restrict access to sensitive data, or when some data should never be stored by New Relic. Read our [obfuscation documentation](/docs/logs/ui-data/obfuscation-ui/) to find out more.
 
+## Details about log forwarding [#forwarding-details]
```

Comment:
my only comment is that most other agents have a way to get the additional keys in a log line by using `application_logging.forwarding.context_data.include` and `application_logging.forwarding.context_data.exclude`